Cargo.toml

```diff
@@ -33,7 +33,7 @@ harness = false
 [features]
 unstable = ["tendril/unstable", "string_cache/unstable"]
 heap_size = ["heapsize", "heapsize_plugin"]
-codegen = ["html5ever_macros"]
+codegen = []
```

Comment:
This feature is now unused, but removing it would be a breaking change. So let’s keep it until we have some other reason to make a breaking change.